### ISSUE TITLE
无法复现COCO上的结果

### ISSUE THREAD

**Issue by user6251** [wuyongjianCODE/SDPT#2]
1. 在COCO上只微调SDPT模块，性能相比于zero-shot 51.4 下降到50.6；
2. attention_mask_l=torch.cat((attention_mask_l,self.cpt_maskpad),dim=1)  导致逆映射的文本prompt失效
